client/src/templates/Challenges/components/Challenge-Description.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react';
+
+import PrismFormatted from './PrismFormatted';
+import './challenge-description.css';
+
+type Challenge = {
+  block: string;
+  description: string;
+  instructions: string;
+};
```

Comment:
At some point we'll likely want to build this out as a full exportable type, independent of component files.

Reply:
I will keep that in mind